### DISCUSSION TITLE
"Refresh" & "Find GPS Location" Toggle

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#116] [Ideas]
A toggle/slider in the settings enabled the pull-down to refresh feature. It also includes finding GPS locations. This allows the user to pull down to refresh and set the GPS location in one swipe.

**Comment by farfromrefug** [comment]
@jacobgonzales20 that is a good idea.
so basically you want pull to refresh to either? :
* refresh weather data
* refresh location then refresh weather

**Reply by ghost** [reply]
> @jacobgonzales20 that is a good idea.
> so basically you want pull to refresh to either? :
> * refresh weather data
> * refresh location then refresh weather 

Correct. This doesn't have to be a toggle, it could just be a change but I figured. I would just propose it as a toggle first

**Reply by farfromrefug** [reply]
@jacobgonzales20 I will make I a setting. myself for example never ever use the gps feature.
